microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1462.37
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1418.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 108.0.1462.37 or higher.
+
+### General
+
+This WebView2 SDK release has the same [bug fixes](#101466-prerelease) that are in WebView2 SDK 1.0.1466-prerelease.

```

Comment:
```suggestion
This WebView2 SDK release has the same [bug fixes](#bug-fixes-1466-prerelease) that are in WebView2 SDK 1.0.1466-prerelease.
```
Tentative new direct link design, would need testing in rendered page.  per
https://learn.microsoft.com/en-us/contribute/how-to-write-links#explicit-anchor-links